PyRDF/backend/Backend.py

```diff
@@ -1,6 +1,7 @@
-from abc import ABC, abstractmethod
+import abc, six
```

Comment:
This is great, I have a couple of comments:
- We can trust the `six` module to be always there, right? If I install Python, it comes with it out of the box?
- Can we do `import six` and `from abc import ABCMeta, abstractmethod`, so that we do not need to prepend `abc` below?

Reply:
@etejedor , I have the same doubt about `six` module. It's available on my local and SWAN (without explicitly installing). But for safer side, I think I'll switch to something else. Here's another workaround for this : [https://gist.github.com/alanjcastonguay/25e4db0edd3534ab732d6ff615ca9fc1](https://gist.github.com/alanjcastonguay/25e4db0edd3534ab732d6ff615ca9fc1)

Reply:
And yes. I'll fix that import 👍

Reply:
@shravan97 @etejedor I think `ABC = abc.ABCMeta('ABC', (object,), {})` is the way to go. `six` is an external python module, so we would need to deal with an external dependency. It is present in SWAN because we do provide it on purpose.

Reply:
Yup. I too feel so @JavierCVilla . Shall I finalize and push the commits @etejedor ?

Reply:
@etejedor and @JavierCVilla , I've pushed the 2 commits now 👍